### DISCUSSION TITLE
Is it possible to change the setter/getter naming conventions to lombok kinda for a Builder class?

### DISCUSSION THREAD

**Discussion by siva54** [kobylynskyi/graphql-java-codegen#921] [Q&A]
After generating my classes, I found out that the Builder classes that are generated have setters with the below structure, 

public Builder setVersion(Long version)

Instead of this, Is it possible to have it as below,

public Builder version(Long version)

This is something the Lombok does today for us.

I have a great part of my code following this way,

builder().version(...).build();

**Comment by IMaleev** [comment]
It would be great to get rid of the builder classes and just add Lombok @Builder annotation instead. Especially if toBuilder attribute is used: @Builder(toBuilder = true).
This way generated classes would be much smaller.

**Comment by jxnu-liguobin** [comment]
I tend to add support for lombok annotations, with existing pojo naming unchanged. To be honest, lombok relies on plugins and can provide support, but it should not be the only choice
